plugins/plugin-svelte/test/unmocked.test.js

```diff
@@ -1,25 +0,0 @@
-const path = require('path');
-const plugin = require('../plugin');
-
-const mockConfig = {buildOptions: {sourceMaps: false}, installOptions: {rollup: {plugins: []}}};
-const mockComponent = path.join(__dirname, 'Button.svelte');
-
-describe('@snowpack/plugin-svelte (unmocked)', () => {
-  it('generates code', async () => {
-    const options = {configFilePath: './plugins/plugin-svelte/test/svelte.config.js'};
-    const sveltePlugin = plugin(mockConfig, options);
-    const result = await sveltePlugin.load({filePath: mockComponent});
-
-    // assume if some CSS & JS were returned, it transformed successfully
-    expect(result['.css'].code).toBeTruthy();
-    expect(result['.js'].code).toBeTruthy();
```

Comment:
When I added this test, I wanted to make sure that the plugin itself always returned a good response. Do we have another test elsewhere (create-snowpack-app snapshots?) that ensures that this plugin always transforms successfully?